rules-topics/01_introduction.md

```diff
@@ -62,6 +62,56 @@ Welcome to Chronicles. Whether you've been LARPing for years, or have never seen
+
+Status that require Identify:
+|Identify Needed|
+|:---:|
+|Animate|
+|Curse|
+|Curse of Transformation|
+|Infect|
+|Spirit Bottle|
+
+### Floating Reset
+Some skills or abilities are listed as "Daily".  These skills and abilities are renewed between 4:00 and 6:00 PM.  If you are on a module between that timeframe you must attempt the reset immediately after a mod.  Daily skills and abilities require 10 minutes of concentration/Rest to refresh.
+
+### How Does Combat Work
+
+Chronicles is a boffer combat Larp that focuses as much on roleplay and interaction between PCs and NPCs as physical combat.  We emphasize light combat, and a general rule is if you swing enough for it to hurt someone you are probably swinging too hard.  The goal of combat is to be safe and while accidents may happen you are more likely to twist an ancle on uneven ground than hurt yourself during combat.  More details can be found in the Combat section of the rulebook.
+
+### Plot Members and Marshals
+
+Plot Members and Marshals are officials of Chronicles.  They will give rulings, run a mod, observe a ritual, and other OOG tasks that are needed throughout a market fair.  If you have a question on a rule or how something works you will need to ask the appropriate marshal.  
+
+Plot Member – A member of the group of people who work together to keep the game moving, adjudicate rules, run mods and many other roles essential to the game.  In table tops terms they are essentially the game master or dungeon master.
+
+Mod Marshal – A person who runs a mod for a group for PCs.  They will set the stage giving descriptions, locations, etc. Should any rule questions or disputes come up, the Mod Martial has final say.  During a mod a Martial rule is final, if you feel they were incorrect it can be adjudicated later when it does not disrupt the flow of the game.
+
+Ritual Marshal – A person who oversees a PC perform ritual magic.  They will ensure all factors of the ritual are performed by the PCs and determine if the ritual is a success or failure.  More complex rituals will require a full plot member.
+
+Rogue Marital – If you wish to steal from someone Cabin you must have a Rogue Martial present.  This individual is to ensure that personal property is not taken and that the would be thief follows all rules of the game (lockpicking chests, recognizing circles of power, etc.)  Entering a cabin with intent to steal without a Rogue Martial can have severe consequences including referring the person to the local police for theft or breaking and entering.
+
+### How to Become A Ritual Marshal
+
+To become a Ritual Marshal you need to volunteer to the Head of Plot.  They will ensure that you know the ritual rules, have you marshal some rituals with a plot member and once you have proven yourself you can become a ritual Marshal. 
```

Comment:
Can we lowercase the last use of "Marshal" here?

Reply:
Fixed